contracts/script/IncredibleSquaringDeployer.s.sol

```diff
@@ -73,7 +67,12 @@ contract IncredibleSquaringDeployer is Script, Utils {
     IIncredibleSquaringTaskManager
         public incredibleSquaringTaskManagerImplementation;
 
+    address public aggregatorAddr;
+    address public taskGeneratorAddr;
+
     function run() external {
+        aggregatorAddr = vm.envAddress("AGGREGATOR_ADDR");
+        taskGeneratorAddr = vm.envAddress("TASK_GENERATOR_ADDR");
```

Comment:
Fetching of the environment variables should go in a setUp() function

Reply:
```solidity
    function setUp() public virtual {
        aggregatorAddress = vm.envAddress("AGGREGATOR_ADDRESS");
        taskGeneratorAddress = vm.envAddress("TASK_GENERATOR_ADDRESS");
    }
```

Reply:
Yeah I agree, can we also move reading the json into the setup function and make the Eigenlayer contracts state variables of the script contract